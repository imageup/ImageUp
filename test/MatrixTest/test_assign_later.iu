function main() -> int
{
	matrix : x(2, 2);
	x = [1, 2 | 3, 4];
	printf(x[0][0]);
	return 0;
}