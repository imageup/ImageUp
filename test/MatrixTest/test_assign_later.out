mat[0][0]: 1.00000
mat[0][1]: 2.00000
mat[1][0]: 3.00000
mat[1][1]: 4.00000
